File: TeeNew/Builder2009/Function_Median.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Function_Median.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Base"
#pragma resource "*.dfm"
TMedianFunctionDemo *MedianFunctionDemo;
//---------------------------------------------------------------------------
__fastcall TMedianFunctionDemo::TMedianFunctionDemo(TComponent* Owner)
  : TBaseForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TMedianFunctionDemo::FormCreate(TObject *Sender)
{
  Series1->FillSampleValues();
}
//---------------------------------------------------------------------------

File: TeeNew/Function_Median.pas
unit Function_Median;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QExtCtrls, QStdCtrls, QComCtrls,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, ExtCtrls, StdCtrls, ComCtrls,
  {$ENDIF}
  Base, TeEngine, TeeFunci, Series, TeeProcs, Chart;

type
  TMedianFunctionDemo = class(TBaseForm)
    Series1: TFastLineSeries;
    Series2: TLineSeries;
    TeeFunction1: TMedianTeeFunction;
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.DFM}
{$ELSE}
{$R *.xfm}
{$ENDIF}

procedure TMedianFunctionDemo.FormCreate(Sender: TObject);
begin
  inherited;
  Series1.FillSampleValues;
end;

initialization
  RegisterClass(TMedianFunctionDemo);
end.
